src/ibusunicode.c

```diff
@@ -608,6 +611,16 @@ ibus_unicode_data_load (const gchar *path,
     return retval;
 }
 
+static void 
+ibus_destroy_list_and_data (gpointer unicode_list) {
```

Comment:
Please delete `ibus_destroy_list_and_data()`. I don't think this internal function is needed.